app/writer_csv.py

```diff
@@ -0,0 +1,45 @@
+import csv
+from dataclasses import astuple, fields, dataclass
+
+
+@dataclass
+class Author:
+    lib = []

```

Comment:
lib: list
set default as the list is so bad practice
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html?highlight=list

Reply:
not fixed